unittests/test_check_filters.py

```diff
@@ -37,37 +38,52 @@ def setUp(self):
                 'num_gpus_per_node': 1})
         ]
 
+
     def test_have_name(self):
         self.assertEqual(1, sn.count(filter(filters.have_name('check1'),
```

Comment:
Just to save same space I would do the following:

```python
def count_checks(self, filter_fn):
    return sn.count(filter(filter_cn, self.checks))

def test_xxx(self):
    self.assertEqual(1, count_checks(filters.have_name('check1')))
```

Reply:
Done.